github/Organization.py

```diff
@@ -596,6 +597,31 @@ def has_in_public_members(self, public_member):
         )
         return status == 204
 
+    def invite_to_organization(self, user, admin=None):
```

Comment:
Can you add `role` has a parameter and assert the values of role to only accept "admin" or "member", and default to "member" if nothing is provided. Also this method does not need to return anything. You can refer the `Team.add_membership()` as an example.